src/WebJobs.Script/Extensions/HttpRequestMessageExtensions.cs

```diff
@@ -70,9 +70,8 @@ public static TValue GetRequestPropertyOrDefault<TValue>(this HttpRequestMessage
 
         public static IDictionary<string, string> GetRawHeaders(this HttpRequestMessage request)
         {
-            Dictionary<string, string> headers = new Dictionary<string, string>();
-
-            var allHeadersRaw = request.Headers.ToString();
+            var headers = new Dictionary<string, string>();
+            var allHeadersRaw = request.Headers.ToString() + request.Content?.Headers?.ToString();
             var rawHeaderLines = allHeadersRaw.Split(new string[] { Environment.NewLine }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
The accept header is separated by commas + space - those are all parsed out the header collection ToString?